### PR TITLE
zh: fix the description about continue profile

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-tidb-operator/pull/1581

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
continue profile 适用于 1.2 吗？

**Comment by KanShiori** [comment]
> continue profile 适用于 1.2 吗？

不适用

File: zh/access-dashboard.md [hunk 1]
```diff
@@ -200,7 +200,9 @@ spec:
 
 启用持续性能分析，你需要使用 v1.3.0 版本及以上的 TiDB Operator 部署 TidbNGMonitoring CR。
 
-1. 参照[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)部署 TidbMonitor CR。
+1. 如果 TiDB 集群版本小于 v5.4.0，参照[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)部署 TidbMonitor CR。
```
Comment (shichun-0415) [inline zh/access-dashboard.md:203]:
```suggestion
1. 部署 TidbMonitor CR。

    - 如果 TiDB 集群版本小于 v5.4.0，参考[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)部署 TidbMonitor CR。
    
    - 如果 TiDB 集群版本大于等于 v5.4.0，可以跳过这一步。
```
如果 TiDB 集群版本大于等于 v5.4.0，是集群自动带有 TidbMonitor CR 还是因为什么呢？感觉交代下直接跳过的原因比较好。

File: zh/access-dashboard.md [hunk 2]
```diff
@@ -200,7 +200,9 @@ spec:
...
+   
+    如果 TiDB 集群版本大于等于 v5.4.0，可以跳过这一步。
```
Comment (shichun-0415) [inline zh/access-dashboard.md:205]:
```suggestion
```

File: zh/access-dashboard.md (hunk 1 above)
Comment (KanShiori) [reply zh/access-dashboard.md:203]: 因为 Dashboard 在 5.4.0 更改了判断能否开启持续分析的逻辑，所以不需要部署 TidbMonitor 了。这个原因感觉对用户来说就是不需要，没啥特殊的原因。

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**